src/authentication/types.ts

```diff
@@ -2,8 +2,7 @@ export const authServerBaseUrlInjectSymbol = Symbol('auth-server-base-url');
 export const gitlabRootPasswordInjectSymbol = Symbol('gitlab-root-password');
 export const jwtOptionsInjectSymbol = Symbol('token-verify-options');
 
-export const subEmailClaimKey: 'https://sub_email' = 'https://sub_email';
-export const teamTokenClaimKey: 'https://team_token' = 'https://team_token';
+export const teamTokenClaimKey: 'https://minard.io/team_token' = 'https://minard.io/team_token';
```

Comment:
Oh, I assumed the type of `teamTokenClaimKey` is implicitly `https://minard.io/team_token` if you assign a const to that string. If that's not the case, then nevermind.